<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Senales_biomedicas</class>
 <widget class="QWidget" name="Senales_biomedicas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>441</width>
     <height>551</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblGraficaSenal">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gráfica señal</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblhistograma">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Histograma</string>
    </property>
   </widget>
   <widget class="QFrame" name="framehistograma">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="framegraficasenal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>361</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblTipoSenal">
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tipo de señal:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comBoxTiposenal">
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>ECG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EEG</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblCanal">
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Canal:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxsenales">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tablesenal">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:0.339795 rgba(255, 0, 0, 255), stop:0.339799 rgba(255, 255, 255, 255), stop:0.662444 rgba(255, 255, 255, 255), stop:0.662469 rgba(0, 0, 255, 255), stop:1 rgba(0, 0, 255, 255));</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCargarSenal">
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>Cargar Señal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Archivo:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblarchivocargadosenal">
      <property name="minimumSize">
       <size>
        <width>329</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>__________________</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGraficarSenal">
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>Graficar Señal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btngraficarhistograma">
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>Graficar Histograma</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblestadoproceso">
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Estado: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblestadosenal">
      <property name="text">
       <string>________________</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnCerrarsenales">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>620</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>VOLVER</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
